dspace-api/src/main/java/org/dspace/util/SimpleMapConverter.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.services.ConfigurationService;
+
+/**
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.it)
+ */
+public class SimpleMapConverter implements SimpleConverter {
+
+    protected String mappingFile; // The properties absolute filename
+
+    protected String converterNameFile; // The properties filename
+
+    protected ConfigurationService configurationService;
+
+    protected Map<String, String> mapping;
+
+    protected String defaultValue = "";
+
+    protected List<String> fieldKeys;
+
+    protected Map<String, String> regexConfig = new HashMap<String, String>();
+
+    public final String REGEX_PREFIX = "regex.";
+
+    public void init() {
```

Comment:
Because this method is complex we should describe what it's loading during initialization...either here in JavaDocs or in the JavaDocs of the class.